 1) The patch changes path to perl interpreter because in Debian
    perl is located at /usr/bin

Index: ejabberd/examples/extauth/check_pass_null.pl
===================================================================
--- ejabberd.orig/examples/extauth/check_pass_null.pl
+++ ejabberd/examples/extauth/check_pass_null.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 use Unix::Syslog qw(:macros :subs);
 
